GraphQL/Navigating schemas, types, fetching requests, using mutations & subscriptions/proposal.md

```diff
@@ -0,0 +1,36 @@
+# TEMPLATE
+
+## :fire: Do not edit this file - copy the template and create your own file.
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+Navigating schemas/types/fetching requests, using mutations/subscriptions
+
+**TOPIC:**    
+GraphQL
+
+**DESCRIPTION (5-7+ sentences):**    
+
+GraphQL, a language for APIs, was first developed by Facebook. After it was introduced, it became extremely popular as some considered it a better API than rest API because underfetching and overfetching data no longer became a problem. GraphQL allows us to specify the root field and the payload from the client side, giving greater flexibility to manipulate the data we want to request from the server. Furthermore, GraphQL has special features of allowing the client to update data, commonly known as mutations, as well as receive real-time data, the concept of subscriptions. These are the core concepts I will be covering within my blog. 
+
+### :pushpin: Step 2
+:family: **TARGET AUDIENCE (3-5+ sentences):**    
+
+My target audience are beginning coders who have little to no experience with GraphQL. I have quite a bit of experience with coding but I currently do not have much knowledge with GraphQL. My goal is to help others get to the level that I will be in after finishing this blog. All in all, this blog will be concise and easy to learn for students who have minimal experience with coding. 
+
+### :pushpin: Step 3
+> Outline your learning/teaching structure: 
+
+**Beginning (2-3+ sentences):**    
+
+As instructed, our blog is supposed to be catered towards people with little to no experience with GraphQL, therefore I will teach my topic in a format that presents a lot of examples. I will begin with giving background information about what GraphQL is: what is it, why is it important, and why do we use it? 
+
+**Middle (2-3+ sentences):**    
+
+Once the topic of GraphQL is covered, I will transition to what mutations and subscriptions are. I am unfamiliar with how to create a client and a server, but once I learn how to do so, I will present code deliverables with images of how to fetch requests related to mutations and subscriptions from the server and client side. 
+
+**End (2-3+ sentences):**    
+
+When the code deliverables are complete, I will explain the next steps that we can take to go further in what we have learned in this blog. I want to encourage others to continue learning GraphQL and work on personal projects of their own to further their knowledge in this topic. 
```

Comment:
Maybe you can also consider what you would've learned from the blog as an experienced coder, since you mention that you would like to know more about GraphSQL.